subprojects/docs/src/docs/release/notes.md

```diff
@@ -272,6 +272,33 @@ There are better ways for re-using task logic, for example by using [task depend
 
 The default output location in [EclipseClasspath](dsl/org.gradle.plugins.ide.eclipse.model.EclipseClasspath.html#org.gradle.plugins.ide.eclipse.model.EclipseClasspath:defaultOutputDir) changed from `${project.projectDir}/bin` to `${project.projectDir}/bin/default`.
 
+### Blacklist repository on `IOException`s 
```

Comment:
:x: I'd say it's rather a optimization than a breaking change. The breaking change is expressed by the next point. I'd list this under "new and noteworthy".

Needs to be more expressive. Maybe "Skip subsequent dependency request upon first failure to repository"?

Reply:
Sure, moved.